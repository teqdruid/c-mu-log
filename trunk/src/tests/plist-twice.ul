foo($y, $u, "hello", $y, 6);
