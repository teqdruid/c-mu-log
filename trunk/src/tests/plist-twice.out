Fatal error: exception Failure("You cannot list the same variable twice in a parameter list")
